doc-cheat-sheet.rst

```diff
@@ -0,0 +1,611 @@
+:orphan:
+
+=========================
+Documentation cheat sheet
+=========================
+
+The documentation files use `reStructuredText (rST) <https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html>`_ syntax. |x wokeignore:rule=master x|
+
+See the following sections for syntax help and conventions.
+
+For general style conventions, see the `Canonical Documentation Style Guide <https://docs.ubuntu.com/styleguide/en>`_.
+
+Headings
+========
+
+.. list-table::
+   :header-rows: 1
+
+   * - Input
+     - Description
+   * - .. code::
+
+          Title
+          =====
+     - Page title and H1 heading
+   * - .. code::
+
+          Heading
+          -------
+     - H2 heading
+   * - .. code::
+
+          Heading
+          ^^^^^^^
+     - H3 heading
+   * - Underlines with other characters, for example, ``"``, ``~``, ``#`` or ``*``
```

Comment:
Do we want to prescribe a fixed order of underline characters or leave it up to each author?

I would go for a fixed order, but don't care which. ;)

Reply:
Fixed order all the way.

I followed upstream OpenStack for H1, H2, and H3 and I followed some other existing OpenStack project (I forget which) for H4. I decided unilaterally on H5 I think.

Reply:
Updated.